### PR TITLE
Fix avatar image updating

### PR DESCRIPTION
**Pull request by osdnk**
https://streamable.com/1cmuj7

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -1029,4 +1029,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 05e9306ad3122d59e620bfaa48069a823cc90577
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.10.0
```
Comment (osdnk) [inline ios/Podfile.lock:1032]:
```suggestion
COCOAPODS: 1.9.1
```

File: src/components/profile/ProfileMasthead.js [hunk 2]
```diff
@@ -108,14 +107,19 @@ export default function ProfileMasthead({
...
-    const account = find(newWallet.addresses, ['address', accountAddress]);
-
-    account.image = null;
-    newWallet.addresses[account.index] = account;
+    const newWallets = {
+      ...wallets,
+      [selectedWallet.id]: {
+        ...wallets[selectedWallet.id],
+        addresses: wallets[selectedWallet.id].addresses.map(account =>
+          account.address === accountAddress
```
Comment (brunobar79) [inline src/components/profile/ProfileMasthead.js:115]: To be safe use toLower on both sides

File: src/components/profile/ProfileMasthead.js [hunk 3]
```diff
@@ -126,15 +130,22 @@ export default function ProfileMasthead({
...
-                : image?.path;
-              dispatch(walletsSetSelected(newWallets[walletId]));
-              return true;
-            });
+            const newWallets = {
+              ...wallets,
+              [selectedWallet.id]: {
+                ...wallets[selectedWallet.id],
+                addresses: wallets[selectedWallet.id].addresses.map(account =>
+                  account.address === accountAddress
```
Comment (brunobar79) [inline src/components/profile/ProfileMasthead.js:138]: Same

**Review by brunobar79 — APPROVED** [review]
💯

**Pull request merged**